contrib/cf-remote/cf_remote/remote.py

```diff
@@ -106,8 +106,9 @@ def get_info(host, *, users=None, connection=None):
 
         tags = []
         if data["os_release"]:
-            distro = data["os_release"]["ID"]
-            major = data["os_release"]["VERSION_ID"].split(".")[0]
+            distro = data["os_release"]["ID_LIKE"] or data["os_release"]["ID"]
```

Comment:
I think this is problematic. Since you prefer `ID_LIKE`, I would expect this to cause cf-remote to download debian packages for ubuntu.